tests/integration/partnerOrganizations.test.js

```diff
@@ -14,9 +13,9 @@ describe('NODE ENV', () => {
 
 describe('PARTNER ORGANIZATION ROUTES - POST /partnerorganizations', () => {
   let authToken;
-  beforeEach(populateDB);
 
   describe('COACH', () => {
+    beforeEach(populateDB);
     beforeEach(async () => {
       authToken = await getToken('coach');
     });
```

Comment:
sur le premier test should add a partner organization peut-être rajouter une vérif que l'organisation a bien été ajoutée

Reply:
done